### PR TITLE
Add command to change joint limits

### PR DESCRIPTION
**Pull request by mpowelson**
Accompanies [Tesseract #415](https://github.com/ros-industrial-consortium/tesseract/pull/415)

### CODE REVIEW

File: tesseract_ros/tesseract_rosutils/src/utils.cpp [hunk 1]
```diff
@@ -1946,18 +1964,19 @@ tesseract::Tesseract::Ptr fromMsg(const tesseract_msgs::Tesseract& tesseract_msg
...
-  if (!tesseract_rosutils::fromMsg(*manip_manager, tesseract_msg.kinematics_information))
-  {
-    ROS_ERROR_STREAM("Failed to populate manipulator manager from kinematics information!");
-    return nullptr;
-  }
+  //  if (!tesseract_rosutils::fromMsg(*manip_manager, tesseract_msg.kinematics_information))
+  //  {
+  //    ROS_ERROR_STREAM("Failed to populate manipulator manager from kinematics information!");
+  //    return nullptr;
+  //  }
```
Comment (mpowelson) [inline tesseract_ros/tesseract_rosutils/src/utils.cpp:1976]: @Levi-Armstrong Where should this go now?

**Comment by mpowelson** [comment]
I think keeping these two commits makes sense

**Comment by Levi-Armstrong** [comment]
Should we squash merge this one?

**Comment by mpowelson** [comment]
Yeah you can if you want.

**Comment by mpowelson** [comment]
Oh wait, are you just talking about the merge commit? I probably should have done a rebase merge. I can force push it if you want

**Comment by Levi-Armstrong** [comment]
Lets do that. Just force push and then we can merge.

**Comment by mpowelson** [comment]
Done

**Review by Levi-Armstrong — APPROVED** [review]

**Pull request merged**